build-config/make/kernel.make

```diff
@@ -96,7 +96,7 @@ LINUX_NEW_FILES	= \
 endif
 
 kernel-build: $(KERNEL_BUILD_STAMP)
-$(KERNEL_BUILD_STAMP): $(KERNEL_SOURCE_STAMP) $(LINUX_NEW_FILES) $(LINUXDIR)/.config | $(XTOOLS_BUILD_STAMP)
+$(KERNEL_BUILD_STAMP): $(KERNEL_SOURCE_STAMP) $(LINUX_NEW_FILES) $(LINUXDIR)/.config
```

Comment:
You can't modify how the kernel is built for all the other machines.

I imagine this is a down stream effect of not building the toolchain locally.